dxr/plugins/js/makefile

```diff
@@ -1,5 +1,5 @@
 build:
-	npm install
+	cd analyze_js; npm install
```

Comment:
We'll need to use npm-lockdown on this before we can deploy it, like we do at DXR's top level. Otherwise, we could get fed any old code from the npm repo.